transport/src/test/java/io/el/channel/DefaultChannelPipelineTest.java

```diff
@@ -70,4 +83,67 @@ public void removeOnlyOne() {
       assertNull(channelPipeline.firstContext());
     }
   }
+
+  @Test
+  public void testBind() {
```

Comment:
We need to mock channel to test the `bind` call.
Netty implemented a new socket class to test the DefaultChannelPipeline

[link](https://github.com/netty/netty/blob/bfd2c5ac8ac83b1983d6f8ae20595e469b2aafbe/transport/src/test/java/io/netty/channel/DefaultChannelPipelineTailTest.java#L250)

Should we follow the method?